fastlane/lib/fastlane/actions/increment_build_number.rb

```diff
@@ -31,7 +31,7 @@ def self.run(params)
         # Attention: This is NOT the version number - but the build number
 
         agv_enabled = system([command_prefix, 'agvtool what-version', command_suffix].join(' '))
-        raise "Apple Generic Versioning is not enabled." unless agv_enabled
+        raise "Apple Generic Versioning is not enabled." unless agv_enabled || params[:build_number]
```

Comment:
Could we possible change this to an `if` instead of an `unless`? I always struggle with `unless` statements that use `||` and `&&` 😝

Reply:
Changed to `raise "Apple Generic Versioning is not enabled." if agv_disabled && params[:build_number].nil?` - hopefully it's a bit more clear 🙂